cfgov/core/utils.py

```diff
@@ -35,3 +35,15 @@ def extract_answers_from_request(request):
     answers = [(param.split('_')[1], value) for param, value in
                request.POST.items() if param.startswith('questionid')]
     return answers
+
+
+def slice_list(l, n):
```

Comment:
Could you add a docstring here describing what this does?  Python has a lot of slicing functions so it'd be good to clarify *how* exactly this slices a list (it's clearer in the tests but not here in the code). I ran it locally with some different inputs hoping that would make it clearer, but unfortunately I'm unable to recreate `list(slice_list([1, 2, 3, 4, 5], 3))` (from your test) because I get a `TypeError` (`'list' object is not callable`).